### DISCUSSION TITLE
Kafka UI Unable to Load Topic Details

### DISCUSSION THREAD

**Discussion by DanukaPraneeth** [provectus/kafka-ui#3828] [Q&A]
Hi All,

I deployed a kafka-UI container in a server and tried to connect with the kafka cluster using the below command.

`docker run -p 8080:8080 --name kafka-ui -e KAFKA_CLUSTERS_0_NAME=local -e KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS=XX.XX.XX.XX:9093 -d provectuslabs/kafka-ui:latest`

Kafka UI is able to load broker details in the web console, but it cannot load the "topic" and "consumer" details. 
![image](https://github.com/provectus/kafka-ui/assets/23314807/1a7d3682-c47d-40d8-9ead-25f2434a8ce4)

I can see the below error in the container logs.

```
2023-05-16 14:36:44,326 INFO  [kafka-admin-client-thread | kafka-ui-admin-1684244881-1] o.a.k.c.NetworkClient: [AdminClient clientId=kafka-ui-admin-1684244881-1] Disconnecting from node 261 due to socket connection setup timeout. The timeout value is 32528 ms.
2023-05-16 14:36:44,846 DEBUG [parallel-33] c.p.k.u.s.ClustersStatisticsScheduler: Metrics updated for cluster: local
2023-05-16 14:36:44,848 DEBUG [parallel-34] c.p.k.u.s.ClustersStatisticsScheduler: Start getting metrics for kafkaCluster: local
2023-05-16 14:36:49,053 ERROR [parallel-35] o.s.b.a.w.r.e.AbstractErrorWebExceptionHandler: [b09cf81d-30]  500 Server Error for HTTP GET "/api/clusters/local/topics?page=1&perPage=25&showInternal=false"
org.apache.kafka.common.errors.TimeoutException: Timed out waiting to send the call. Call: listOffsets on broker 261
	Suppressed: reactor.core.publisher.FluxOnAssembly$OnAssemblyException: 
Error has been observed at the following site(s):
	*__checkpoint ⇢ Handler com.provectus.kafka.ui.controller.TopicsController#getTopics(String, Integer, Integer, Boolean, String, TopicColumnsToSortDTO, SortOrderDTO, ServerWebExchange) [DispatcherHandler]
	*__checkpoint ⇢ com.provectus.kafka.ui.config.CustomWebFilter [DefaultWebFilterChain]
	*__checkpoint ⇢ com.provectus.kafka.ui.config.ReadOnlyModeFilter [DefaultWebFilterChain]
	*__checkpoint ⇢ AuthorizationWebFilter [DefaultWebFilterChain]
	*__checkpoint ⇢ ExceptionTranslationWebFilter [DefaultWebFilterChain]
	*__checkpoint ⇢ LogoutWebFilter [DefaultWebFilterChain]
	*__checkpoint ⇢ ServerRequestCacheWebFilter [DefaultWebFilterChain]
	*__checkpoint ⇢ SecurityContextServerWebExchangeWebFilter [DefaultWebFilterChain]
	*__checkpoint ⇢ ReactorContextWebFilter [DefaultWebFilterChain]
	*__checkpoint ⇢ HttpHeaderWriterWebFilter [DefaultWebFilterChain]
	*__checkpoint ⇢ ServerWebExchangeReactorContextWebFilter [DefaultWebFilterChain]
	*__checkpoint ⇢ org.springframework.security.web.server.WebFilterChainProxy [DefaultWebFilterChain]
	*__checkpoint ⇢ org.springframework.web.filter.reactive.ServerHttpObservationFilter [DefaultWebFilterChain]
	*__checkpoint ⇢ HTTP GET "/api/clusters/local/topics?page=1&perPage=25&showInternal=false" [ExceptionHandlingWebHandler]
```

![image](https://github.com/provectus/kafka-ui/assets/23314807/dca623bd-2c3e-4494-929c-5977822e4ac5)

When I tried the kafka topic list command manually with the same boostrap endpoint used in the above command, it give all the topics. This confirms that the kafka cluster is running without any issues.

Appreciate your inputs.

Thanks!

**Comment by Haarolean** [comment]
@DanukaPraneeth hi, which kafka is that?

**Reply by danukalg** [reply]
Hi @Haarolean,

I am trying to connect with **_confluentinc/cp-kafka:5.5.10_** kafka cluster from my provectuslabs/kafka-ui:latest container.

Thanks!

**Reply by Haarolean** [reply]
@danukalg do you have over 260 brokers?

**Reply by danukalg** [reply]
@Haarolean, I only have 12 brokers.

**Reply by Haarolean** [reply]
Uhm, is @DanukaPraneeth your alt account? If not, please don't hijack others' threads, raise up a new discussion with your info.

**Reply by DanukaPraneeth** [reply]
@Haarolean, sorry for the confusion. My previous response is from an alt work account. I only have 12 brokers in the kafka cluster.

**Reply by Haarolean** [reply]
@danukalg @DanukaPraneeth sorry for the delay, I'll convert this to an issue

**Comment by Haarolean** [comment]
#3958
